Userland/Utilities/jail-create.cpp

```diff
@@ -13,15 +13,21 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     StringView new_jail_name;
     Core::ArgsParser args_parser;
+    bool pid_isolation = false;
+    bool custom_isolation_options = false;
     args_parser.add_positional_argument(new_jail_name, "New jail name", "jail name");
+    args_parser.add_option(custom_isolation_options, "Create jail with custom isolation options", "custom-isolation-options", 'c');
+    args_parser.add_option(pid_isolation, "Use PID-isolation (as a custom isolation option)", "pid-isolation", 'p');
     args_parser.parse(arguments);
 
     TRY(Core::System::pledge("stdio jail"));
 
-    if (!new_jail_name.is_null() && !new_jail_name.is_empty()) {
-        TRY(Core::System::create_jail(new_jail_name));
-        return 0;
-    }
+    if (new_jail_name.is_null() || new_jail_name.is_empty())
+        return Error::from_string_view("Can't create a jail with empty name."sv);
 
-    return Error::from_string_view("Can't create a jail with empty name."sv);
+    if (custom_isolation_options)
+        TRY(Core::System::create_jail_with_custom_isolation_options(new_jail_name, pid_isolation));
+    else
+        TRY(Core::System::create_jail_with_default_isolation_options(new_jail_name));
```

Comment:
Instead of splitting all these methods into two versions ("default" and "custom options"), why not have a single method that takes the isolation flags and let userland (in this case, `jail-create`) determine what flags should be used?

Reply:
But then there's no meaning to the default flag, because having a function:
```
ErrorOr<u64> create_jail(StringView jail_name, bool default_isolation_options, bool pid_isolation)
```
does not make sense. I now just think about dropping the whole concept of "default flag", because it makes things too much complicated.